crates/wasmparser/src/readers/operators.rs

```diff
@@ -76,6 +82,13 @@ impl<'a> OperatorsReader<'a> {
         let pos = self.reader.original_position();
         Ok((self.read()?, pos))
     }
+
+    pub fn range(&self) -> Range {
```

Comment:
We shouldn't need to add `offset` and `data` to `OperatorsReader`. The `BinaryReader` has a `range` method that we can delegate too, and it already keeps track of that offset information internally.

Reply:
Do you mean making the reader field publicly available?

Reply:
I just mean returning `self.reader.range()` instead of keeping track of `offset` and `data` and doing the range calculation ourselves here